index.yaml

```diff
@@ -1,3 +1,181 @@
 apiVersion: v1
-entries: {}
-generated: "2022-03-07T15:26:07.809711909-05:00"
+entries:
+  cluster-openstack:
+  - annotations:
+      application.giantswarm.io/metadata: https://giantswarm.github.io/giantswarm-catalog/cluster-openstack-0.7.0.tgz-meta/main.yaml
+      application.giantswarm.io/readme: https://giantswarm.github.io/giantswarm-catalog/cluster-openstack-0.7.0.tgz-meta/README.md
```

Comment:
Thomas should be `giantswarm-catalog` translated to `cluster-catalog`?

Reply:
I thought I fixed all of those. Good catch.

Reply:
For some reason even when I run `helm repo index .` it keeps the old values. I used find-and-replace, hopefully this works.